packages/web/pages/api/stripe/webhook.ts

```diff
@@ -0,0 +1,107 @@
+import { MembershipSubscriptionPeriod } from '@journaly/j-db-client'
+import stripe from '../../../nexus/utils/stripe'
+import { getClient } from '../../../nexus/utils'
+
+const handler = async (req: any, res: any) => {
+  const db = getClient()
+  const event = req.body
+
+  const updateStripeSubscription = async (subscriptionId: string) => {
```

Comment:
This whole func can be lifted up out of `handler` and get `db` passed in

Reply:
Excellent idea! Done! ✅